test/simple/test-tls-securepair-fiftharg.js

```diff
@@ -0,0 +1,63 @@
+// Copyright Joyent, Inc. and other Node contributors.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a
+// copy of this software and associated documentation files (the
+// "Software"), to deal in the Software without restriction, including
+// without limitation the rights to use, copy, modify, merge, publish,
+// distribute, sublicense, and/or sell copies of the Software, and to permit
+// persons to whom the Software is furnished to do so, subject to the
+// following conditions:
+//
+// The above copyright notice and this permission notice shall be included
+// in all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
+// OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+// MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN
+// NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR
+// OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE
+// USE OR OTHER DEALINGS IN THE SOFTWARE.
+//
+
+var common = require('../common');
+var assert = require('assert');
+var fs = require('fs');
+var tls = require('tls');
+
+test1();
+
+// simple/test-tls-securepair-client
+function test1() {
+    var sslcontext = tls.createSecureContext({
+        cert: fs.readFileSync(common.fixturesDir + '/test_cert.pem'),
+        key: fs.readFileSync(common.fixturesDir + '/test_key.pem'),
+    });
+    var catched_servername = null;
+    var pair = tls.createSecurePair(sslcontext, true, false, false, {
+        SNICallback: function(servername, cb) {
+            catched_servername = servername;
+        }
+    });
+
+    // captured trafic from browser's request to https://www.google.com
+    var ssl_hello_with_SNI = new Buffer([
+        22,3,1,2,0,1,0,1,252,3,3,123,36,32,232,194,245,125,208,188,197,238,164,123,101,37,21,189,4,223,219,197,179,76,
```

Comment:
Sorry, but we use 80 column limit everywhere. I may only suggest to convert this to hex and split it before it gets to 80 column. It will be much more compact this way.